api/swagger-spec/api.json

```diff
@@ -0,0 +1,27 @@
+{
+  "swaggerVersion": "1.2",
+  "apiVersion": "",
+  "basePath": "https://10.192.210.63:8443",
```

Comment:
> Basically we don't really support --master=127.0.0.1 which means we can't support --listen=127.0.0.1

I didn't realize this was a problem.  @liggitt did you know about this?  We don't have any validation preventing this.